sign-in.txt

```diff
@@ -11,4 +11,4 @@ TODO: Use the terminal and git commands to complete the PULL; MERGE; COMMIT; PUS
   NAME  : GitHub Username
   -----------------------------
   Brian : Bansenauer-Cascadia
-  Isaak : isaakwheeler-Cascadiagit
+  Isaak : isaakwheeler-Cascadia
```

Comment:
Isaak,

You did part of it, but missed D-5, that is, leaving all the other names in the list. Give this another try

> D-5.Make any needed edits using Android Studio so that the sign-in list is complete and correct, that is, all of the previous names from develop are still there, with your info is added below them all

Reply:
I'm doing something wrong so I will ask after class or something